src/lib/tracking.ts

```diff
@@ -37,4 +37,4 @@ export const getTrackingUrl = (baseUrl: string, params: EpicTracking): string =>
 const campaignPrefix = 'gdnwb_copts_memco';
 
 export const buildCampaignCode = (test: Test, variant: Variant): string =>
-    `${campaignPrefix}_${test.name}_${variant.name}`;
+    `${campaignPrefix}_${test.campaignId || test.name}_${variant.name}`;
```

Comment:
Hardcoded tests may have their own custom `campaignId` value so we should use that first, if available.

Reply:
Unless I've misunderstood Frontend logic:
- Hardcoded tests all have a dedicated `campaignId` prop, passed as an argument into `makeEpicABTest` where it's passed downstream without changes; this is often, but not always, the test ID with different casing.
- Configured tests don't have `campaignId` in the JSON, instead this is added in `buildConfiguredEpicTestFromJson` and assigned the value of `test.name` (this step also adds those hardcoded-specific meta props to configured tests such as author and description, albeit with dummy values). 

So all tests have a `campaignId`, which is inferred for configured tests but explicit for hardcoded ones.